dex/multi_token_farm/src/rewards.rs

```diff
@@ -1,7 +1,7 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
 
-type Nonce = u64;
+use common_structs::Nonce;
```

Comment:
Just to make sure: this file is deprecated and it is not under the scope of review

Reply:
Yes